test/unit/app/common/state/tabStateTest.js

```diff
@@ -483,6 +483,10 @@ describe('tabState unit tests', function () {
       const expectedAppState = defaultAppState.set('tabs', tabsWithoutField)
       assert.deepEqual(newAppState, expectedAppState)
     })
+    it('returns the state (unchanged) if tabs is falsey', function () {
+      const newAppState = tabState.removeTabField(defaultAppState, 'loginRequiredDetail')
```

Comment:
Fixed- that variable exists in other files (with other states- whoops!). Thanks for catching 😄